content/local/store_unix.go

```diff
@@ -1,4 +1,4 @@
-// +build linux solaris darwin freebsd
+// +build linux solaris darwin freebsd netbsd 
```

Comment:
there is a trailing space on this line, causing the DCO/git whitespace check to fail, and maybe the reason `goimports` is compaining?

Reply:
Whoops, nice catch @estesp!

I have just removed the extra blank.

Thanks Phil!